content/en/docs/tasks/administer-cluster/network-policy-provider/antrea-network-policy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Use Antrea for NetworkPolicy
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+This page shows how to create an Antrea cluster on Kubernetes.
+For background on Project Antrea, read the [Introduction to Antrea](https://antrea.io/docs/).
+
+## {{% heading "prerequisites" %}}
+
+You need a Kubernetes cluster created on premisses or in Public Cloud, and the kubectl command-line tool must be configured to communicate with your cluster.
+
+<!-- steps -->
+
+## Deploying Antrea with kubeadm
+
+When using `kubeadm` to create the Kubernetes cluster, passing
+`--pod-network-cidr=<CIDR Range for Pods>` to `kubeadm init` will enable
+`NodeIpamController`. Clusters created with kubeadm will always have
+`CNI` plugins enabled. Refer to
+[Creating a cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm)
+for more information about setting up a Kubernetes cluster with `kubeadm`.
+
+To deploy the latest version of Antrea (built from the master branch), use the
+checked-in [deployment yaml](https://github.com/vmware-tanzu/antrea/blob/master/build/yamls/antrea.yml):
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea.yml
+```
+
+## Deploying Antrea in Kind
+
+To deploy Antrea in a [Kind](https://github.com/kubernetes-sigs/kind) cluster,
+please refer to this [guide](https://github.com/vmware-tanzu/antrea/blob/master/docs/kind.md).
+
+## Deploying Antrea in EKS and GKE
+
+Antrea can be deployed in NetworkPolicy only mode to an EKS cluster or a GKE
+cluster, and enforce NetworkPolicies for the cluster.
+
+* To deploy Antrea in an EKS cluster, please refer to [the EKS installation guide](https://github.com/vmware-tanzu/antrea/blob/master/docs/eks-installation.md).
+* To deploy Antrea in a GKE cluster, please refer to [the GKE installation guide](https://github.com/vmware-tanzu/antrea/blob/master/docs/gke-installation.md).
+
+## Deploying Antrea with IPsec Encryption
```

Comment:
i think it is worth while adding a reference to the Antrea on Windows config, which also provides a NetworkPolicy implementation for Windows